pre_commit_hooks/ros_include_guard.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2022 Tier IV, Inc. All rights reserved.
```

Comment:
How about adding `Black` and `isort` to `.pre-commit-config.yaml`?

Reply:
Since it will change `sort_package_xml.py`, making it another PR is better.

Reply:
I believe @KeisukeShima can help with this task.
@isamu-takagi Please talk to him if you need some help!

Reply:
Did it in #33.